src/ReduxedStorage.ts

```diff
@@ -79,7 +79,7 @@ export default class ReduxedStorage<
         mergeOrReplace(this.state, state) : mergeOrReplace(state, this.state);
       !newTime && isEqual(newState, this.state) ||
         ( this._setState(newState, timestamp), this._renewStore() );
-      newTime && isEqual(newState, state) ||
+      if (newTime && !isEqual(newState, state))
```

Comment:
Does the current logic preclude sending to storage when the new state is not equal? If so, does that mean the current logic is wrong?

Reply:
Actually, it **does** sending when the new state is not equal

Reply:
https://www.loom.com/share/c1ddd0bd78144160aaefdc224cbd6d4e

Reply:
Well, no matter old or new code/logic, the qualifying condition is the same - inequality (aside from `newTime` condition). It seems you haven't noticed OR operator in the old code which negates the previous expression. In other words:
```
newTime && isEqual(newState, state) || something
```
is equivalent to:
```
if (!newTime || !isEqual(newState, state)) something
```

Reply:
I think we might be talking about 2 different things, but I do understand what you're saying now!